client/app/visualizations/table/renderer.less

```diff
@@ -0,0 +1,101 @@
+@import (reference, less) "~@/assets/less/ant.less";
+
+.table-visualization-container {
+  .ant-pagination.ant-table-pagination {
+    float: none;
+    display: block;
+    text-align: center;
+    margin-bottom: 0;
+  }
+
+  .ant-table-body {
+    overflow-x: auto;
+  }
+
+  table {
+    border-top: @border-width-base @border-style-base @border-color-split;
+    border-bottom: 0;
+
+    tbody tr:last-child td {
+      border-bottom: 0;
+    }
+
+    thead {
+      .anticon.off {
+        opacity: 0;
+      }
+
+      &:hover .anticon.off {
+        opacity: 1;
+      }
+    }
+
+    .display-as-number,
+    .display-as-boolean,
+    .display-as-datetime,
+    .display-as-image {
+      width: 1%;
+      white-space: nowrap;
+    }
+
+    .table-visualization-spacer {
+      padding-left: 0;
+      padding-right: 0;
+
+      & > div:before {
+        content: '' !important;
+      }
+    }
+
+    th {
+      & > div:before {
+        content: '\00a0'; // &nbsp; - to preserve column height
+        display: inline-block;
+        min-width: 50px; // min-width of column contents
+      }
+
+      .ant-input-search {
+        font-weight: normal;
+
+        .ant-input-suffix .anticon {
+          cursor: auto;
+        }
+      }
+
+      .table-visualization-heading {
+        display: inline-block;
+        white-space: nowrap;
+        position: absolute;
```

Comment:
This means that the column would always be as wide as it's content - never it's title and imho it makes the th ellipsised too frequently.

I was thinking of simply giving it a max-width. Sth like:
```less
.table-visualization-heading {
  // nothing

  span {
    display: inline-block;
    max-width: 100%;
    vertical-align: middle;
  }
}

.ant-table-column-sorter {
  right: -3px;
}
.ant-table-column-has-sorters {
  padding-right: 12px !important;
}
```

### Before (lots of ellipsis):

<img width="716" alt="Screen Shot 2019-07-31 at 7 45 11" src="https://user-images.githubusercontent.com/486954/62184531-36913480-b367-11e9-9cf4-542a045a186f.png">

### After (not as much):

<img width="716" alt="Screen Shot 2019-07-31 at 7 44 21" src="https://user-images.githubusercontent.com/486954/62184532-36913480-b367-11e9-837c-e74d5ebe5a9e.png">


Wdyt?

Reply:
Yeah, it's way much better:
![image](https://user-images.githubusercontent.com/12139186/62207647-50e60500-b39d-11e9-90a5-b0bf4a6d32a6.png)